src/neo4j_graphrag/experimental/components/entity_relation_extractor.py

```diff
@@ -414,34 +311,43 @@ async def run(
         self,
         chunks: TextChunks,
         document_info: Optional[DocumentInfo] = None,
+        lexical_graph_config: Optional[LexicalGraphConfig] = None,
         schema: Union[SchemaConfig, None] = None,
         examples: str = "",
         **kwargs: Any,
     ) -> Neo4jGraph:
-        """Perform entity and relation extraction for all chunks in a list."""
+        """Perform entity and relation extraction for all chunks in a list.
```

Comment:
Could we define 'lexical graph' either here or in the user guide?

Reply:
This is what I tried to do [here](https://github.com/neo4j/neo4j-graphrag-python/pull/127/files#diff-c37850688512e77f403fe5c0f70c11f569a5ae233777f9859877f2f49dfb4619R179-R184). What is missing? I'll add a note in this docstring anyway to clarify the behavior of `create_lexical_graph` and `lexical_graph_config`.

Reply:
Perhaps how is the term 'lexical graph' from any other knowledge graph. I think a newcomer will be confused with that

Reply:
Added more details and link to the user guide, let me know if you think more information is needd.